olp-cpp-sdk-core/CMakeLists.txt

```diff
@@ -138,12 +138,10 @@ file(GLOB NETWORK_SOURCE
     "${CMAKE_CURRENT_SOURCE_DIR}/src/network/*.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/network/*.h"
 )
-set(NETWORK2_SOURCE
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/network2/NetworkProxySettings.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/network2/NetworkRequest.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/network2/NetworkResponse.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/network2/NetworkSettings.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src/network2/Network.cpp"
+
+file(GLOB EDGE_SDK_HTTP_SOURCES
```

Comment:
Why would you switch to GLOBing sources here, but then not for EDGE_SDK_HTTP_HEADERS?